tests/bootstrap.php

```diff
@@ -9,6 +9,8 @@
  * file that was distributed with this source code.
  */
 
+error_reporting(E_ALL | E_STRICT);
```

Comment:
Not sure about this line. I just added this to make sure the notice for getTestNodeClass() will be thrown when running phpunit.